scripts/deploy-storagenode.sh

```diff
@@ -6,7 +6,7 @@ set -euo pipefail
 : "${TAG:?Must be set to the gitish version of the release without architecture}"
 
 for v in alpha arm beta; do
-	docker manifest create storjlabs/storagenode:$v \
+	docker manifest create --amend storjlabs/storagenode:$v \
 	storjlabs/storagenode:${TAG}-amd64 \
```

Comment:
As far as I understood the name of the images in `docker manifest` is this one and the rest listed below.

These images names don't change, so I'm wondering if we are using the same images for `alpha` and `beta`?

On the other hand, what `arm` means in a list of versions, when this is building images for different architectures?

Reply:
I don't understand the magic myself. Don't ask me^^